docker-compose-common.yml

```diff
@@ -25,6 +25,7 @@ services:
             - ./lti/:/app/lti
             - ./pdfservice/:/app/pdfservice
             - ./static/src/:/app/static/src
+            - ./webpack-stats/:/app/webpack-stats
```

Comment:
Nice fix! I often forget about the webpack stats.

Reply:
(ditto! thanks!)